app/src/main/java/com/example/jondhc/yoyo/Level1Activity.java

```diff
@@ -1,23 +1,288 @@
+                            basket_berry.setVisibility(View.VISIBLE);
+                            break;
+                        case 8:
+                            audio = R.raw.eight;
+                            basket_berry = findViewById(R.id.berry_basket_8);
+                            basket_berry.setVisibility(View.VISIBLE);
+                            break;
+                        case 9:
+                            audio = R.raw.nine;
+                            basket_berry = findViewById(R.id.berry_basket_9);
+                            basket_berry.setVisibility(View.VISIBLE);
+                            break;
+                    }//end switch
+                    MediaPlayer mp = MediaPlayer.create(getApplicationContext(), audio);    //set the audio to selected audio
+                    mp.start(); //Play the audio of the number of fruits.
+                    if (basket_counter == quantity) {   //When all the fruits are collected
+                        Timer timer = new Timer();  //Timer for Winning audio to run
+                        timer.schedule(new userWon(), 1000);    //Setting delay for audio
+                        System.out.println("User won!");
+                        //Animation of character
+                        char_cat = findViewById(R.id.cat_catch);    //Getting character image
+                        RotateAnimation char_anim = new RotateAnimation(0f, 360f, char_cat.getWidth() / 2, char_cat.getHeight() / 2);
+                        char_anim.setInterpolator(new LinearInterpolator());
+                        char_anim.setRepeatCount(0);
+                        char_anim.setDuration(700);
+                        char_cat.startAnimation(char_anim);
+                        //ends Animation of character
+                    }//end if
+                    return true;
+                case DragEvent.ACTION_DRAG_ENDED:
+                    if (event.getResult()) {    //When drop action was made in the selected place
+                        System.out.println("Was dropped in the correct place");
+                        View originView = (View) event.getLocalState(); //Get the element that was dropped
+                        originView.setVisibility(View.INVISIBLE);   //If the fruit was dropped correctly, set as invisible
+                    } else if (!event.getResult()) {    //When drop action was not succesful (fruit dropped outside the basket)
+                        System.out.println("Fruit was dropped outside");
+                        View originView = (View) event.getLocalState(); //Get the element that was dropped
+                        originView.setAlpha(1); //return to its initial state
+                    }//end else if
+                    break;
+                default:
+                    break;
+            }//end switch
+            return true;
+        }//end onDrag
+    }//end DragHandlingListener
+}//end Level1Activity
```

Comment:
I think it'd be interesting for this level to perform a loop (when the kid dragged all the fruits, new fruits appears), like 3 times. So, several numbers would also be covered